sdk/ml/azure-ai-ml/tests/component/unittests/test_command_component_entity.py

```diff
@@ -453,7 +450,8 @@ def test_primitive_output(self):
             component1._to_rest_object().as_dict()["properties"]["component_spec"], *omits
         )
 
-        assert actual_component_dict1 == expected_rest_component
+        print(actual_component_dict1)
+        assert actual_component_dict1 == expected_rest_component, actual_component_dict1
```

Comment:
why are we doing "a == b, c" here?

Reply:
Updated this for print c originally, which may better show what is the difference; however, c here is too long, the output is not ideal at all, then I add print here.

This change is redundant now and I forget to remove this; already removed this in my new commit.